tests/test_F0_output.py

```diff
@@ -0,0 +1,206 @@
+        F0.store_timeseries_all_busses_to_excel(dict_timeseries_test_two_busses)
+        test = False
+        if (
+            os.path.exists(os.path.join(OUTPUT_PATH, "timeseries_all_busses" + ".xlsx"))
+            is True
+        ):
+            if (
+                os.path.exists(os.path.join(OUTPUT_PATH, "a_bus" + " flows.png"))
+                is True
+            ):
+                if (
+                    os.path.exists(os.path.join(OUTPUT_PATH, "b_bus" + " flows.png"))
+                    is True
+                ):
+                    test = True
+
+        assert test is True
+
+    def test_store_dict_into_json(self):
+        file_name = "test_json_converter"
+        F0.store_as_json(json_test_dictionary, OUTPUT_PATH, file_name)
+        assert os.path.exists(os.path.join(OUTPUT_PATH, file_name + ".json")) is True
+
+    def teardown_module(self):
+        if os.path.exists(OUTPUT_PATH):
+            shutil.rmtree(OUTPUT_PATH, ignore_errors=True)
+
+
+class TestDictionaryToJsonConversion:
+    if not os.path.exists(OUTPUT_PATH):
+        os.mkdir(OUTPUT_PATH)
+
+    def test_processing_dict_for_json_export_parse_bool(self):
+        file_name = "test_json_bool"
+        F0.store_as_json(json_test_dictionary["bool"], OUTPUT_PATH, file_name)
+        assert os.path.exists(os.path.join(OUTPUT_PATH, file_name + ".json")) is True
+
+    def test_processing_dict_for_json_export_parse_str(self):
+        file_name = "test_json_str"
+        F0.store_as_json(json_test_dictionary["str"], OUTPUT_PATH, file_name)
+        assert os.path.exists(os.path.join(OUTPUT_PATH, file_name + ".json")) is True
+
+    def test_processing_dict_for_json_export_parse_numpy_int64(self):
+        expr = F0.convert(json_test_dictionary["numpy_int64"])
+        assert expr == scalar
+
+    def test_processing_dict_for_json_export_parse_pandas_DatetimeIndex(self):
+        expr = F0.convert(json_test_dictionary["pandas_DatetimeIndex"])
+        assert expr == "date_range"
+
+    def test_processing_dict_for_json_export_parse_pandas_Timestamp(self):
+        expr = F0.convert(json_test_dictionary["pandas_Timestamp"])
+        test = False
+        if expr == start_time:
+            test = True
+        assert test == False
+
+    def test_processing_dict_for_json_export_parse_pandas_series(self):
+        expr = F0.convert(json_test_dictionary["pandas_series"])
+        assert expr == "pandas timeseries"
+
+    def test_processing_dict_for_json_export_parse_numpy_array(self):
+        expr = F0.convert(json_test_dictionary["numpy_array"])
+        assert expr == "numpy timeseries"
+
+    def test_processing_dict_for_json_export_parse_pandas_Dataframe(self):
+        expr = F0.convert(json_test_dictionary["pandas_Dataframe"])
+        assert expr == "pandas dataframe"
+
+    def test_processing_dict_for_json_export_parse_unknown(self):
+        with pytest.raises(TypeError):
+            F0.convert(unknown_type)
+
+    def teardown_module(self):
```

Comment:
```suggestion
    def teardown_method(self):
```
Assuming you want to to perform it on every method of the class, otherwise `teardown_class`